util/bazelisk.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```

Comment:
Arguably this file is so important it should live at the top of the repo, so people write `./bazelisk.sh`.

I am low-key a fan of the name of Rust's cursed make/cargo wrapper, `x.py`, so maybe we should go to the extreme and name it `./bz.sh`.

Reply:
Named `bazelisk.sh` in t he root of the project.